allennlp/models/encoder_decoders/nlvr_semantic_parser.py

```diff
@@ -66,6 +69,11 @@ class NlvrSemanticParser(Model):
         Mixture weight (0-1) for combining coverage cost and denotation cost. As this increases, we
         weigh the coverage cost higher, with a value of 1.0 meaning that we do not care about
         denotation accuracy.
+    dynamic_cost_weight : ``Dict[str, Union[int, float]]``
+        A dict containing keys ``wait_num_steps`` and ``rate`` indicating the number of steps after
+        which we should start decreasing the weight on checklist cost in favor of denotation cost,
+        and the rate at which we should do it. The weight will remain constant if this is an empty
```

Comment:
Giving an equation here would be nice, because "the rate at which we should do it" can mean lots of things.

Reply:
Done.